### PR TITLE
TW-1383: Improve edit group

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket 
- [x] #1383
- [x] #1279 

### Resolved 

https://github.com/linagora/twake-on-matrix/assets/99852347/42ffe5e9-12d2-4992-8867-9eb24e3ddfe0

https://github.com/linagora/twake-on-matrix/assets/99852347/c281d113-74c4-424e-bc7e-919c30f16593

### CODE REVIEW

File: lib/pages/chat_details/chat_details_edit.dart [hunk 1]
```diff
@@ -90,7 +93,11 @@ class ChatDetailsEditController extends State<ChatDetailsEdit>
   ) {
     final listAction = [
       EditChatAvatarContextMenuActions.edit,
-      EditChatAvatarContextMenuActions.delete,
+      if ((room?.avatar != null ||
+              avatarFilePicker != null ||
+              avatarAssetEntity != null) &&
+          !_isDeleteAvatar)
```
Comment (imGok) [inline lib/pages/chat_details/chat_details_edit.dart:99]: could you define a variable for this pls

File: lib/pages/chat_details/chat_details_edit.dart [hunk 2]
```diff
@@ -404,12 +411,15 @@ class ChatDetailsEditController extends State<ChatDetailsEdit>
...
         }
       },
-      (success) {
+      (success) async {
         if (success is UpdateAvatarGroupChatSuccess) {
           Logs().d(
             'ChatDetailsEdit::_handleUpdateGroupInfoOnEvents() - UpdateAvatarGroupChatSuccess',
           );
           _handleSaveIconOnUpdateSuccess();
+          await Future.delayed(_delayedUpdateAvatarDuration).then((_) {
```
Comment (imGok) [inline lib/pages/chat_details/chat_details_edit.dart:420]: why is the delay needed?

File: lib/pages/chat_details/chat_details_edit.dart (hunk 1 above)
Comment (hoangdat) [reply lib/pages/chat_details/chat_details_edit.dart:99]: A method

Comment (nqhhdev) [reply lib/pages/chat_details/chat_details_edit.dart]: Done

File: lib/pages/chat_details/chat_details_edit.dart (hunk 2 above)
Comment (nqhhdev) [reply lib/pages/chat_details/chat_details_edit.dart:419]: I don't know exactly avatar update successfully in `room`. So I wait a little time for it.

Screenshot 2024-02-01 at 08 45 15

**Review by sherlockvn — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**